sources/packages/backend/apps/api/src/testHelpers/fake-entities/program-year-fake.ts

```diff
@@ -1,12 +1,21 @@
 import { ProgramYear } from "@sims/sims-db";
 
-export function createFakeProgramYear(): ProgramYear {
+export function createFakeProgramYear(
```

Comment:
The `createFakeProgramYear` is created by default under the lib because they are potentially used everywhere. There is already a `createFakeProgramYear` at the libs that can be replaced entirely by this code (that looks way better 😉 ).